Rubberduck.VBEEditor/SafeComWrappers/VB6/VBComponents.cs

```diff
@@ -196,7 +196,7 @@ public void RemoveSafely(IVBComponent component)
         private bool _eventsAttached;
         private void AttachEvents()
         {
-            throw new NotImplementedException("Correct the Guid (see comment above), verify the DispIds, then remove this throw.");
+           // throw new NotImplementedException("Correct the Guid (see comment above), verify the DispIds, then remove this throw.");
```

Comment:
**Never** comment-out a `throw new NotImplementedException` call. Either it's implemented and should be removed, or it isn't, and it throws a NotImplementedException accordingly. This otherwise becomes ambiguously dead code.

And why is there a diff on a VB6 COM wrapper anyway?

Reply:
Not my module. This came down from my Rubberduck fork.

Reply:
hmm, that might have been my commit - I don't remember commenting it, but will fix it up on my next VB6 commits.